src/glib/base/ut.h

```diff
@@ -190,6 +190,31 @@ class TLogNotify : public TNotify {
 	void OnStatus(const TStr& MsgStr);
 };
 
+//////////////////////////////////////
+// Color-Notify
+enum class TColorNotifyType {
```

Comment:
Requires consistent number of indentation spaces. The rest of the code has 4 spaces.